test/integration/roles-permissions-test-cases.yaml

```diff
@@ -0,0 +1,49 @@
+config:
+  inherit-env: true
+
+tests:
+  001 - list role permissions:
+    command: auth0 roles permissions ls $(./test/integration/scripts/get-role-id.sh)
```

Comment:
Should we add a test where we assign a permission before hand (via script) and the assert the output? We can keep this one as well as it asserts the empty state.

Reply:
My bad, I just saw test `004` 🤦🏼‍♀️